addon/templates/components/frost-tab.hbs

```diff
@@ -1,10 +1,13 @@
 <div class="tab {{if disabled 'disabled'}}"
-  data-test={{hook (concat hook '-tab') selected=isSelected}}>
+  data-test={{hook (concat hook '-tab') selected=isSelected}}
+>
   {{frost-button
     class=(concat 'tab' (if isSelected ' active'))
     disabled=disabled
+    hook=(concat hook '-tabBtn')
```

Comment:
I'd prefer `-tabButton`, I know it's longer, but I'm not a fan of abbreviations in general